bluepysnap/network.py

```diff
@@ -144,13 +144,13 @@ def _get_ids_from_pop(self, fun_to_apply, returned_ids_cls, sample=None, limit=N
         return res
 
     @abc.abstractmethod
-    def ids(self, group=None, sample=None, limit=None):
+    def ids(self, group=None, sample=None, limit=None, node_sets=None):
         """Resolves the ids of the NetworkObject."""
 
     @abc.abstractmethod
-    def get(self, group=None, properties=None):
+    def get(self, group=None, properties=None, node_sets=None):  # pylint: disable=too-many-locals
```

Comment:
I can try to get this with less locals in this PR, or we can review it when we make change the get in NSETM-2186.